samples/contrib/kubeflow-katib/mpi-job-horovod.py

```diff
@@ -32,10 +32,9 @@
     name="Launch Katib MPIJob Experiment",
     description="An example to launch Katib Experiment with MPIJob"
 )
-def horovod_mnist_hpo():
-    # Experiment name and namespace.
-    experiment_name = "mpi-horovod-mnist"
-    experiment_namespace = "anonymous"
+def horovod_mnist_hpo(
+        experiment_name = "mpi-horovod-mnist",
```

Comment:
How about this formatting:
```python
def horovod_mnist_hpo(
    experiment_name: str = "mpi-horovod-mnist",
    experiment_namespace: str = "anonymous",
):
```

Reply:
I agree with this, since this is followed at other places as well.